data/colors/vars/deprecated_light.ts

```diff
@@ -542,9 +542,7 @@ export default {
       numHoverText: get('fg.default')
     },
     deletion: {
-      numHoverText: get('fg.default'),
-      lineBg: get('danger.subtle'),
-      wordBg: get('danger.muted')
```

Comment:
We should remove `deletion.lineBg` and `deletion.wordBg` from `deprecated_dark.ts` as well. We also need to make sure that they're not in the `deprecations.json`.

Reply:
Done!